src/js/site/insightsTableScroll.js

```diff
@@ -0,0 +1,23 @@
+// This function will highlight a dom element with the ID found in the URL fragment
+// Added to draw attention to specific entries in only the insights table
+// Highlight can be generalized to rest of documentation later see main.scss
+export function insightsTableScroll() {
```

Comment:
This function could be used to run on any article / page. Can we rename this to something more general?

Reply:
Renamed to highlightURLHash, since it actually doesn't do any scrolling only highlighting